### PR TITLE
enable null geometries for deletions in /contributions

### LINKED ISSUE #129: invalid geojson returned for deletions on contributions endpoint
Deleted elements are represented by `null` geometries in normal `contributions/geometry` requests. However, when using the `contributions/centroid` endpoint (and perhaps also `…/bbox`), one gets the following invalid GeoJSON result instead:

```json
  …
  {
    "type" : "Feature",
    "geometry" : {
      "type" : "Point",
      "coordinates" : [ ]
    },
    "properties" : {
      "@changesetId" : 84648597,
      "@deletion" : "true",
      "@osmId" : "way/800143465",
      "@osmType" : "WAY",
      "@timestamp" : "2020-05-04T21:23:50Z",
      "@version" : 2
    }
  }
  …
```

image

//edit: this is how I would have expected as a result:

```json
  {
    "type" : "Feature",
    "geometry" : null,
    "properties" : {…}
  }
```

### PR DESCRIPTION
**Pull request by FabiKo117**
### Description
use null for the geometry instead of an empty coordinate array within the response GeoJSON feature for deletions

### Corresponding issue
Closes #129

### Checklist
- [x] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md#code-style) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/)~
- [x] I have commented my code
- ~~I have written javadoc (required for public methods)~~
- [x] I have added sufficient unit and API tests
- [x] I have made corresponding changes to the [documentation](https://github.com/GIScience/ohsome-api/tree/master/docs)
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-api/blob/master/CHANGELOG.md)
- ~~I have adjusted the examples in the [check-ohsome-api](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api) script or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api/-/issues/new) in the corresponding repository. More Information [here](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md#check-examples).~~

### CODE REVIEW

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java [hunk 1]
```diff
@@ -412,28 +403,28 @@ public void writeCsvResponse(Result[] resultSet, HttpServletResponse servletResp
...
-          return new org.wololo.geojson.Feature(emptyPolygon, properties);
+          return new org.wololo.geojson.Feature(null, properties);
         }
         if (deletionHandling && geometry.getGeometryType().contains("LineString")) {
-          return new org.wololo.geojson.Feature(emptyLine, properties);
+          return new org.wololo.geojson.Feature(null, properties);
         }
         if (deletionHandling && geometry.getGeometryType().contains("Point")) {
-          return new org.wololo.geojson.Feature(emptyPoint, properties);
+          return new org.wololo.geojson.Feature(null, properties);
```
Comment (tyrasd) [inline src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java:427]: I don't see the point of the repeated `if` cluster here to always return the same result. Also, `GeometryCollections` fall through here

**Review by tyrasd — COMMENTED** [review]
did you consider fixing #131 in this PR as well?

File: src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java (hunk 1 above)
Comment (tyrasd) [reply src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java:427]: PS: the `if (deletionHandling) { … }` could even be performed before the `switch (elemGeom)` statement, right?

Comment (FabiKo117) [reply src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java]: Yes, you're right. Added that now as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Changelog
 ### Bug Fixes
 
 * fix some invalid filters in the default swagger examples ([#111])
+* fix returning invalid GeoJSON using empty coordinates for deletion contributions ([#129])
```
Comment (tyrasd) [inline CHANGELOG.md:9]: please mention #131 also in the changelog

**Review by tyrasd — CHANGES_REQUESTED** [review]
would it be possible to also add a test case for the `GeometryCollection` case (for #131)?

**Review by tyrasd — APPROVED** [review]

**Pull request merged**